src/components/TransactionsList.js

```diff
@@ -1,38 +1,79 @@
-import React from 'react'
-
-const TransactionsList = () => {
-
-  return (
-    <table className="ui celled striped padded table">
-      <tbody>
-        <tr>
-          <th>
-            <h3 className="ui center aligned header">
-              Posted At
-            </h3>
-          </th>
-          <th>
-            <h3 className="ui center aligned header">
-              Description
-            </h3>
-          </th>
-          <th>
-            <h3 className="ui center aligned header">
-              Category
-            </h3>
-          </th>
-          <th>
-            <h3 className="ui center aligned header">
-              Amount
-            </h3>
-          </th>
-        </tr>
-
-        {"... your code here..."}
-
-      </tbody>
-    </table>
-  )
-}
+import React, { Component } from 'react'
+
+export default class TransactionsList extends Component {
```

Comment:
We discussed this in the review, but this component doesn't need to have state. You want your state to be in the highest possible place. That way you don't have to hook into `componentWillReceiveProps`